lib/tasks/index.rake

```diff
@@ -5,6 +5,106 @@ require "octokit"
 require "time"
 require "digest/sha1"
 
+
+def index_l10n_doc(filter_tags, doc_list, get_content)
+
+  ActiveRecord::Base.logger.level = Logger::WARN
+  rebuild = ENV["REBUILD_DOC"]
+  rerun = ENV["RERUN"] || rebuild || false
+
+  filter_tags.call(rebuild, false).sort_by { |tag| Version.version_to_num(tag.first[1..-1]) }.each do |tag|
+    name, commit_sha, tree_sha, ts = tag
+    puts "#{name}: #{ts}, #{commit_sha[0, 8]}, #{tree_sha[0, 8]}"
+
+    stag = Version.where(name: name.gsub("v", "l10n")).first_or_create
+
+    next if (stag.commit_sha == commit_sha) && !rerun
+
+    stag.commit_sha = commit_sha
+    stag.tree_sha = tree_sha
+    stag.committed = ts
+    stag.save
+
+    tag_files = doc_list.call(tree_sha)
+    doc_files = tag_files.select { |ent| ent.first =~
+        /^([_\w]+)\/(
+          (
+            git.*
+        )\.txt)/x
+    }
+
+    puts "Found #{doc_files.size} entries"
+    doc_limit = ENV["ONLY_BUILD_DOC"]
+
+    get_content_f = Proc.new do |source, target|
+      name = File.join(File.dirname(source), target)
+        content_file = tag_files.detect { |ent| ent.first == name }
+        if content_file
+          new_content = get_content.call (content_file.second)
+        else
+          puts "can not resolve #{name}\n"
```

Comment:
what does this means? is it some unexpected error? asking because I'm getting a few while running your instructions. (which means it's not probably obvious as well to future new users)

Reply:
When translating content, some source files may not be translated yet. During their inclusion in other files, they are missing, so the resulting document will miss the corresponding content. This is a warning. The enclosing content is present, so it's a best effort scenario where we provide as much as possible what's already available. Another option would be to reject the manpage if any included part is missing, but that would hide a lot of already done work.

This is a copy of what's already done for the original manpages, but as expected for these ones, there are never unresolved content.